compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/PackageLoader.java

```diff
@@ -283,14 +290,15 @@ public BLangPackage loadEntryPackage(PackageID pkgId, PackageID enclPackageId, b
         return packageNode;
     }
 
-    public BLangPackage loadPackage(PackageID pkgId, PackageID enclPackageId, PackageRepository packageRepo) {
+    public BLangPackage loadPackage(PackageID pkgId, PackageID enclPackageId, RepoHierarchy encPkgRepoHierarchy) {
         // TODO Remove this method()
         BLangPackage bLangPackage = packageCache.get(pkgId);
         if (bLangPackage != null) {
             return bLangPackage;
         }
 
-        BLangPackage packageNode = loadPackageFromEntity(pkgId, loadPackageEntity(pkgId, enclPackageId));
+        BLangPackage packageNode = loadPackageFromEntity(pkgId, loadPackageEntity(pkgId, enclPackageId,
+                                                                                                encPkgRepoHierarchy));
```

Comment:
Shall we reformat this code?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10442/commits/fa74255bc087ed97ca3d7fb2f28539751a69e5f0